tests/foreman/api/test_subscription.py

```diff
@@ -31,273 +31,280 @@
+        upload_manifest(orgs[0].id, manifest.content)
+        with pytest.raises(TaskFailedError):
+            upload_manifest(orgs[1].id, manifest.content)
+    assert len(entities.Subscription(organization=orgs[1]).search()) == 0
+
+
+@pytest.mark.tier2
+def test_positive_delete_manifest_as_another_user(function_org):
+    """Verify that uploaded manifest if visible and deletable
+        by a different user than the one who uploaded it
+
+    :id: 4861bdbc-785a-436d-98cf-13cfef7d6907
+
+    :expectedresults: manifest is refreshed
+
+    :BZ: 1669241
+
+    :CaseImportance: Medium
+    """
+    user1_password = gen_string('alphanumeric')
+    user1 = entities.User(
+        admin=True,
+        password=user1_password,
+        organization=[function_org],
+        default_organization=function_org,
+    ).create()
+    sc1 = ServerConfig(
+        auth=(user1.login, user1_password),
+        url=f'https://{settings.server.hostname}',
+        verify=False,
+    )
+    user2_password = gen_string('alphanumeric')
+    user2 = entities.User(
+        admin=True,
+        password=user2_password,
+        organization=[function_org],
+        default_organization=function_org,
+    ).create()
+    sc2 = ServerConfig(
+        auth=(user2.login, user2_password),
+        url=f'https://{settings.server.hostname}',
+        verify=False,
+    )
+    # use the first admin to upload a manifest
+    with manifests.clone() as manifest:
+        entities.Subscription(sc1, organization=function_org).upload(
+            data={'organization_id': function_org.id}, files={'content': manifest.content}
         )
-        # use the first admin to upload a manifest
-        with manifests.clone() as manifest:
-            entities.Subscription(sc1, organization=org).upload(
-                data={'organization_id': org.id}, files={'content': manifest.content}
-            )
-        # try to search and delete the manifest with another admin
-        entities.Subscription(sc2, organization=org).delete_manifest(
-            data={'organization_id': org.id}
+    # try to search and delete the manifest with another admin
+    entities.Subscription(sc2, organization=function_org).delete_manifest(
+        data={'organization_id': function_org.id}
+    )
+    assert len(Subscription.list({'organization-id': function_org.id})) == 0
+
+
+@pytest.mark.tier2
+def test_positive_subscription_status_disabled(module_ak, rhel7_contenthost, module_org):
+    """Verify that Content host Subscription status is set to 'Disabled'
+     for a golden ticket manifest
+
+    :id: d7d7e20a-e386-43d5-9619-da933aa06694
+
+    :expectedresults: subscription status is 'Disabled'
+
+    :BZ: 1789924
+
+    :CaseImportance: Medium
+    """
+    rhel7_contenthost.install_katello_ca()
+    rhel7_contenthost.register_contenthost(module_org.label, module_ak.name)
+    assert rhel7_contenthost.subscribed
+    host = entities.Host().search(query={'search': f'name={rhel7_contenthost.hostname}'})
```

Comment:
```suggestion
    host = rhel7_contenthost.nailgun_host
```

Reply:
golfed this down even further, using that new property though.